src/App.js

```diff
@@ -234,7 +247,10 @@ class App extends Component {
     );
 
     try {
-      const [/*blockhash*/, feeCalculator] = await this.connection.getRecentBlockhash();
+      const [
```

Comment:
@mvines this is an unanticipated lint change, does it bother you?

Reply:
meh oh well :)